yt/data_objects/construction_data_containers.py

```diff
@@ -2183,8 +2183,8 @@ def __init__(self, left_edge=None, right_edge=None, n_ref=32, over_refine_factor
         YTSelectionContainer3D.__init__(self,
             center, ds, field_parameters)
 
-        self.left_edge = self._sanitize_edge(left_edge)
-        self.right_edge = self._sanitize_edge(right_edge)
+        self.left_edge = self._sanitize_edge(left_edge, ds.domain_left_edge)
```

Comment:
This is a behavior change, right?  Now you can not specify left and right edges to this object?  This is an improvement -- I am just asking a clarifying question.

Reply:
Not sure what you're talking about here. You can still specify that.